website/docs/Using-Premake.md

```diff
@@ -47,7 +47,7 @@ premake5 --help
 
 ## Using the Generated Projects
 
-For toolsets like Visual Studio and Xcode, you can simply load the generated workspace or workspace into your IDE and build as you normally would.
+For toolsets like Visual Studio and Xcode, you can simply load the generated workspace or workspaces into your IDE and build as you normally would.
```

Comment:
This originally said `solution or workspace` and during a mass rename of `solution` to `workspace` this weirdness was created.

@starkos I feel like this should be reverted back to `solution or workspace` since those are what is generated, thoughts?

Reply:
Agreed, "solution or workspace" was the intended text there. @abhiss Would you be able/willing to adjust your PR?

Reply:
> Agreed, "solution or workspace" was the intended text there. @abhiss Would you be able/willing to adjust your PR?
Did it. Do I need to merge changes from master?